reference/clever-tools/networkgroups.md

```diff
@@ -0,0 +1,192 @@
+
+> **Disclaimer:** It might not be up-to-date. Please warn us or <TODO: explain how to contribute on GitHub> if you see anything that's changed.
+
+**General tips:**
+
+- `clever ng` is an alias for `clever networkgroups`. I'll use it to make commands more readable.
+- Command arguments order do not matter, but to have similar commands, I used the convention of placing `--ng NG_ID_OR_LABEL` just after `clever ng`.
+- You can add `--help` or `-?` to print help about a command usage.
+- You can add `--verbose` or `-v` to run commands in verbose mode if you need more details about what's happening under the hood (for debug purposes for example).
+
+### Create a networkgroup
+
+To create a networkgroup, you need to provide a label and a description. The label is <TODO: Explain why it cannot contain spaces> and the description is just for you to remember what the networkgroup is about.
+
+An example usage would be:
+
+```sh
+clever ng create --label 'first-ng' --description 'My first networkgroup'
+```
+
+```log
+Networkgroup 'first-ng' was created with the id 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'.
+```
+
+**Tips:**
+
+- You can add `--tags tag1,tag2,tag3` to provide tags for networkgroups management.
+- [Use `--interactive` to answer optional questions](#use---interactive-to-answer-optional-questions)
+
+### List created networkgroups
+
+To check the list of networkgroups you created, you can run:
+
+```sh
+clever ng list
+```
+
+After [Create a networkgroup](#create-a-networkgroup), you should get:
+
+```log
+Networkgroup ID                           Label                 Members  Peers  Description
+────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb   first-ng              0        0      My first networkgroup
+```
+
+**Tip:** [`--json` or `-j` shows output as JSON](#--json-or--j-shows-output-as-json)
+
+### Add a member
+
+After [creating a networkgroup](#create-a-networkgroup), you can add members to it. A member is an app, an add-on or an [external node category](TODO: Redirect to doc). You can set `--type` to `'application'`, `'addon'` or `'external'` according to your needs.
+
+For example, to add an app to the networkgroup, you can do:
+
+```sh
+clever ng --ng 'first-ng' members add --member-id 'app_d60ef23d-80d7-4095-b8d7-7f42bea137ea' --type 'application' --domain-name 'my-app-1' --label 'My app 1'
+```
+
+```log
+Successfully added member 'app_d60ef23d-80d7-4095-b8d7-7f42bea137ea' to networkgroup 'ng_bcce10f3-ec6a-4d51-8e28-50cd622d2ecb'.
+```
+
+**Tips:**
+
+- [`--ng` takes a networkgroup label or ID](#--ng-takes-a-networkgroup-label-or-id)
+- `--label` is optional, but it's a good practice to add it.
+
+### List members in a networkgroup
+
+To list the members in a networkgroup, run `clever ng members list`. For example, after ["Add a member"](#add-a-member), you could run:
```

Comment:
To list members *of* a networkgroup.

After *adding* a member.

This line should be split in two.